### ISSUE TITLE
No docs for ComPtr

### ISSUE THREAD

**Issue by Manishearth** [retep998/wio-rs#19]
https://docs.rs/wio/0.2.0/wio/com/struct.ComPtr.html doesn't exist, perhaps the crate needs some `doc(cfg)` stuff?

**Comment by retep998** [comment]
That is the docs for the linux version. I just need to specify the appropriate docs.rs stuff so it defaults to Windows.

**Comment by Manishearth** [comment]
Yeah, that's what I mean, you can use `doc(cfg)` to make that work IIRC

**Comment by agashlin** [comment]
It's also worthwhile specifying it as the default in Cargo.toml:
```
[package.metadata.docs.rs]
default-target = "x86_64-pc-windows-msvc"
```

Edit: To be clear, this is what retep998 is proposing.
